### PR TITLE
Fix/svelte sdk

### PR DESCRIPTION
**Pull request by samijaber**
## Description

Depends on #1024 

- Fixes Svelte SDK: adds workaround for empty HTML elements
- SolidJS SDK: tweak mitosis.config to work with latest Mitosis version

### CODE REVIEW

File: packages/sdks/src/components/render-block/render-block.lite.tsx [hunk 1]
```diff
@@ -91,20 +92,29 @@ export default function RenderBlock(props: RenderBlockProps) {
...
-            style={state.css}
-          >
-            <For each={state.children}>
-              {(child) => <RenderBlock key={child.id} block={child} />}
-            </For>
-          </state.componentRef>
+          /**
+           * Svelte is super finicky, and does not allow an empty HTML element (e.g. `img`) to have logic inside of it,
+           * _even_ if that logic ends up not rendering anything.
+           */
```
Comment (samijaber) [inline packages/sdks/src/components/render-block/render-block.lite.tsx:116]: 😭

Comment (mrkoreye) [reply packages/sdks/src/components/render-block/render-block.lite.tsx:116]: 😞

File: packages/sdks/src/components/render-block/render-block.lite.tsx [hunk 2]
```diff
@@ -91,20 +92,29 @@ export default function RenderBlock(props: RenderBlockProps) {
...
+        >
+          <For each={state.children}>
+            {(child) => <RenderBlock key={child.id} block={child} />}
+          </For>
+        </state.componentRef>
+      }
+    >
       <Show
-        when={!state.componentInfo?.noWrap}
+        when={!isEmptyHtmlElement(state.tagName)}
```
Comment (mrkoreye) [inline packages/sdks/src/components/render-block/render-block.lite.tsx:111]: No need to check for `noWrap` anymore?

**Review by mrkoreye — APPROVED** [review]

File: packages/sdks/src/components/render-block/render-block.lite.tsx (hunk 2 above)
Comment (samijaber) [reply packages/sdks/src/components/render-block/render-block.lite.tsx:111]: What I actually did is _add_ another nested `Show` statement, not alter the existing one. If you look at the diff with whitespace ignored: https://github.com/BuilderIO/builder/pull/1032/files/c6dbd9537c5e6e42a3a3c8cdd180686b20adfef?w=1

image


it should be easier to parse and see 🤞🏽

Comment (mrkoreye) [reply packages/sdks/src/components/render-block/render-block.lite.tsx:111]: ah yes! i missed that

**Pull request merged**